pgmpy/readwrite/BIF.py

```diff
@@ -0,0 +1,564 @@
+from pgmpy.models import BayesianModel
+
+from pgmpy.factors import TabularCPD
+
+from pgmpy.extern.six.moves import map, range
+
+import numpy
+
+from pyparsing import Word, alphanums, Suppress, Optional, CharsNotIn, Group, nums, ZeroOrMore, OneOrMore,\
+        cppStyleComment, Literal, printables
+
+import re
+
+from string import Template
```

Comment:
@khalibartan I didn't mean to put blank lines after each import. You basically have to categorize the imports into 3 categories (check PEP8) and have a blank line in between each of these three blocks (check other modules).